.gitignore

```diff
@@ -1,3 +1,9 @@
 node_modules
 build
 .DS_Store
+
+# Ignore tree-sitter generated files
+tree-sitter.json
+tree-sitter-vhs.pc
+libtree-sitter-vhs.dylib
+libtree-sitter-vhs.a
```

Comment:
Missing final line break.

Reply:
oh actually that was made by editorconfig plugin, is the rule without or with linebreak @caarlos0 @aymanbagabas @bashbunni ?

Reply:
> oh actually that was made by editorconfig plugin,

that's using the root editorconfig